fe/fe-core/src/main/java/com/starrocks/analysis/OutFileClause.java

```diff
@@ -69,6 +82,45 @@ public class OutFileClause implements ParseNode {
         PARQUET_COMPRESSION_TYPE_MAP.put("default", TCompressionType.DEFAULT_COMPRESSION);
     }
 
+    public static final Set<PrimitiveType> PARQUET_SUPPORTED_PRIMITIVE_TYPES = ImmutableSet.of(
+        PrimitiveType.BOOLEAN,
+        PrimitiveType.TINYINT,
+        PrimitiveType.SMALLINT,
+        PrimitiveType.INT,
+        PrimitiveType.BIGINT,
+        PrimitiveType.FLOAT,
+        PrimitiveType.DOUBLE,
+        PrimitiveType.DATE,
+        PrimitiveType.DATETIME,
+        PrimitiveType.CHAR,
+        PrimitiveType.VARCHAR,
+        PrimitiveType.DECIMAL32,
+        PrimitiveType.DECIMAL64,
+        PrimitiveType.DECIMAL128
+    );
+
+    public static final Set<PrimitiveType> CSV_SUPPORTED_PRIMITIVE_TYPES = ImmutableSet.of(
```

Comment:
Have you test if each type is supported

Reply:
CSV supported types are listed in `csv/converter.cpp`